src/pages/gen1/[platform]/tools/console/adminui/access-management/index.mdx

```diff
@@ -114,3 +114,104 @@ If your Studio application experiences any issues logging in or the resources ha
 If you receive an error that you're not authorized to perform an action, your policies must be updated to allow you to perform the action.
 
 If you need help, contact your AWS administrator. Your administrator is the person who provided you with your sign-in credentials. See [AWS managed policies for AWS Amplify](https://docs.aws.amazon.com/amplify/latest/userguide/security-iam-awsmanpol.html) for more details.
+
+### Updating Login Cognito Lambda triggers runtime
+
+If you need to update the [Cognito Lambda triggers](/gen1/[platform]/tools/console/adminui/access-management/#cognito-lambda-triggers) runtime, you can do so by updating the Lambda functions triggers associated with the Cognito User Pool named **`amplify_backend_manager_<app-id>`**.
+
+1. Log in to the AWS Management Console, open the Amplify console and select your Amplify project with Amplify Studio enabled.
+2. Retrieve the App-ID present on the overview page under the App name.
+3. Go to Amazon Cognito, select `User pools`. Search for the user pool with the App-Id. You will observe the app user pool with the naming format `amplify_backend_manager_<app-id>`.
+4. Click on the user pool and go to `User Pool Properties` which should display the `Lambda Triggers`. 
```

Comment:
styling nit on the "click" usage https://github.com/aws-amplify/docs/blob/main/STYLEGUIDE.md#15-use-carets-to-help-users-navigate-menus-use-select-instead-of-click

for each of the mentions of a noun or place you're directing folks they should be bolded, and reserve inline code blocks for mentions of API's, file names, etc.